mysql-test/suite/rocksdb.rpl/t/suite.opt

```diff
@@ -1 +1 @@
-$ROCKSDB_OPT $ROCKSDB_LOAD_ADD
+$ROCKSDB_OPT $ROCKSDB_LOAD_ADD --loose-rocksdb_unsafe_for_binlog=true
```

Comment:
That doesn't sound safe. Perhaps some of the existing tests need stricter --source include/have_binlog_format_*.inc instead?